website/project/model.py

```diff
@@ -1527,6 +1556,9 @@ def save(self, *args, **kwargs):
                 save=True,
             )
 
+            if self.creator and settings.ENABLE_NOTIFICATION_SUBSCRIPTION_CREATION:
```

Comment:
Just curious: why is it necessary to check if `self.creator`? Looks like we're already assuming that `self.creator` exists here: https://github.com/CenterForOpenScience/osf.io/blob/0512b299781bdfbb89044075f499495c7795129c/website/project/model.py#L1507-L1510